### PR TITLE
1.15.2 Profiler mappings

### PR DESCRIPTION
**Pull request by liach**
Missing stuff and (mainly) 3 new classes

Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/util/profiler/ProfileLocationInfo.mapping [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_4748 net/minecraft/util/profiler/ProfileLocationInfo
+	COMMENT Profiling information on a specific profiler location.
```
Comment (Pyrofab) [inline mappings/net/minecraft/util/profiler/ProfileLocationInfo.mapping:2]: Wouldn't this be a `section` of the code rather than a precise location ?

**Comment by liach** [comment]
@pyrofab we have been using location as it's indicated by a path like a.b.c (replace . with ascii char 30), which location is a better term than section

**Comment by Pyrofab** [comment]
If it is represented by a path, why not use `path` instead of `location` ? A "code path" makes quite a bit of sense too.

**Comment by liach** [comment]
not gonna refactor on that. Path leads to a location, and it's a location in profiler tree not in code, so location makes sense. please don't bother about this!

**Review by modmuss50 — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Pull request merged**